corehq/apps/domain/management/commands/unlink_project_spaces.py

```diff
@@ -0,0 +1,84 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+from django.core.management.base import BaseCommand
+
+from corehq.apps.domain.models import Domain
+from corehq.apps.linked_domain.models import DomainLink
+from corehq.apps.app_manager.models import Application
+from six.moves import input
+
+
+class Command(BaseCommand):
+    help = "Unlinks linked project spaces and converts the downstream app into a standalone app."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'master_domain_name',
+            help='The name of the master project space'
+        )
+        # parser.add_argument(
+        #     'master_app_name',
+        #     help='The name of the master app'
+        # )
+        parser.add_argument(
+            'linked_domain_name',
+            help='The name of the linked project space'
+        )
+        parser.add_argument(
+            'linked_app_name',
+            help='the name of the linked app'
+        )
```

Comment:
I'm not sure app name is precise enough - they're not guaranteed to be unique on the domain